### PR TITLE
adding articles for feb second week

### PR DESCRIPTION
**Pull request by chennakeshava1998**
Added posts from aditya prabhu(evolution of cycles), sachin sampath(why should we buy electric cars) and myself(t-SNE - data visualization tool).

### CODE REVIEW

File: blog_src/_data/authors.yml [hunk 1]
```diff
@@ -525,4 +525,8 @@ Shruti-Deshpande:
   github: Shruti-Deshpande
   author_image: Shruti-Deshpande.jpeg
   user_description: 'Do or Do not. There Is no Try.'
-  email: shrutideshpande9598@gmail.com
\ No newline at end of file
+  email: shrutideshpande9598@gmail.com
+adithyaprabhu98:
```
Comment (salman-bhai) [inline blog_src/_data/authors.yml]: Are you sure his name wasn't there already!

File: blog_src/_posts/2018-02-10-Electric-Cars.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    diesel .
+
+3.  Emissions : If you are an Indian you would definitely have
+    experienced the nauseating puff of smoke being thrown out by the
+    vehicles in front you on the street .With EVs we can finally say
+    goodbye to that and breathe in much fresher air on the streets .
+
+Now the cons,
+
+1.Limited range :Electric engines are the kings of limited mileage range
```
Comment (salman-bhai) [inline blog_src/_posts/2018-02-10-Electric-Cars.md]: Space after the `1. `

File: blog_src/_posts/2018-02-10-tSNE.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+author_github: chennakeshava1998
+date: 2018-02-15 23:32:44
+image: '/assets/img/'
+description: 'Description of t-SNE algorithm'
+tags:
+- CompSoc
+- t-SNE
+- Data Visualization
+categories:
+- ML
```
Comment (salman-bhai) [inline blog_src/_posts/2018-02-10-tSNE.md]: Put CompSoc in categories! Remove the others. You can put it in tags!

File: blog_src/_posts/2018-02-10-tSNE.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+__PCA__ tries to retain the maximum information about high-dimensional data even after embedding it into a low dimension. It does this by finding out a few components in the higher dimension along which the maximum variation of the data is ensured. Hence it preserves the variation in the data even after reducing it to a lower dimension. Mathematically, PCA minimises the squared-error between points - essentially ensures that far-away points in the original dataset remain in the same way in the 2D map also. So PCA preserves large pairwise distances(global structure and properties), but these distances do not imply any meaningful property.
+But it loses the the low-variance deviations between neighbours. t-SNE filled this gap by accounting for local properties, which have a lot of significance irrespective of the overall structure/topology of data.
+
+
+Moreover PCA is not very useful for unlabelled data. This is quite evident by looking at the visualizations produced by PCA and t-SNE on the MNIST dataset. 
+
+![](/blog/assets/img/t-SNE/pca_mnist.png)
```
Comment (salman-bhai) [inline blog_src/_posts/2018-02-10-tSNE.md]: Put some alternate text over here!

File: blog_src/_posts/2018-02-10-tSNE.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+Distinct clusters are not produced by PCA. The distinctness is evident only because of the color-coding, so PCA is not very helpful for unlabelled data.
+
+![](/blog/assets/img/t-SNE/tsne_mnist.jpeg)
```
Comment (salman-bhai) [inline blog_src/_posts/2018-02-10-tSNE.md]: Put some alternate text here!

File: blog_src/_posts/2018-02-16-Evolution-of-bicycle.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+development until date has been in the field of bicycle.
+
+Here are a few of the designs discussed briefly, innovation in them and the
+reasons for their failure or upgradation.
+
+The first known bicycle was the **velocipede**, a human powered vehicle which
+required the person riding it to actually push himself against the ground in
+order to move. No pedals, chain whatsoever, just 2 wheels, a frame and a seat.
+
+![https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/Draisine_or_Laufmaschine%2C_around_1820._Archetype_of_the_Bicycle._Pic_01.jpg/800px-Draisine_or_Laufmaschine%2C_around_1820._Archetype_of_the_Bicycle._Pic_01.jpg](media/e1074583fe8f083cc2b7e8326019e8f9.jpg)
```
Comment (salman-bhai) [inline blog_src/_posts/2018-02-16-Evolution-of-bicycle.md]: I think this file path is wrong!

File: blog_src/_data/authors.yml (hunk 1 above)
Comment (chennakeshava1998) [reply blog_src/_data/authors.yml]: I searched for variants of aditya, adithya, prabhu....in authors.yml but I couldn't find any?

File: blog_src/_posts/2018-02-16-Evolution-of-bicycle.md (hunk 6 above)
Comment (chennakeshava1998) [reply blog_src/_posts/2018-02-16-Evolution-of-bicycle.md]: the jekyll serve worked properly in my computer, so I thought of leaving it as it is. I have changed it now

**Comment by chennakeshava1998** [comment]
included the suggested changes except the change for authors.yml

**Pull request merged**